lib/public/view.js

```diff
@@ -59,6 +60,7 @@ export default (model) => {
 
         'run-overview': RunsOverviewPage,
         'run-detail': RunDetailsPage,
+        'runs-per-period': RunsPerPeriodPage,
```

Comment:
Why is it `runs-per-period` and not LHC periods?

Reply:
renamed